react/Popover/Popover.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+export default class Popover extends Component {
+  constructor(props) {
+    super(props)
+    this.element = ''
+    this.state = {
+      style: { display: 'block', visibility: 'hidden', position: 'fixed' }
```

Comment:
Position fixed has some problems in Safari mobile when scrolling, is it necessary ?